Cartfile.resolved

```diff
@@ -1,8 +1,8 @@
-github "amplitude/Amplitude-iOS" "a3d6720626e72a678489bce83ed9dacadb505dd7"
+github "amplitude/Amplitude-iOS" "32d36977147d9f9a6330002afdc85fe31cfbcfaa"
```

Comment:
Yes – go ahead and check the framework bundles in. I'll squash when I merge, so there's no concern in committing multiple updates if necessary if you want to do so before you cut the release tag (but please run `carthage update rileylink_ios` one last time after the tag is created)